src/components/MapLabel.svelte

```diff
@@ -15,7 +20,14 @@
       <Fa icon={faXmark} />
     </button>
     <div class="map-name">{name}</div>
-    <MapStyleInputWrapper {index} />
+    <div class="options-container">
+      <MapStyleInputWrapper {index} />
+      {#if !$linkLocationsStore && Object.keys(mapState).length}
```

Comment:
Honestly we should only need to check for `#if !$linkLocationsStore` but I added an extra safety condition in case some new bug crops up. That said, it might just be cleaner to remove `Object.keys(mapState).length`